src/po/Make_mvc.mak

```diff
@@ -542,7 +542,7 @@ install-all: all
 	for %%l in ($(LANGUAGES)) do @$(CP) %%l.mo \
 		$(VIMRUNTIME)\lang\%%l\LC_MESSAGES\$(PACKAGE).mo
 
-cleanup-po: $(LANGUAGE).po
+cleanup-po: checklanguage $(LANGUAGE).po
 	$(VIM) -u NONE -e -X -S cleanup.vim -c wq $**
```

Comment:
When processing this rule, an error occurs:
```
"checklanguage" 0 lines, 0 bytes
"checklanguage" 0 lines, 0 bytes written
Error detected while processing command line:
E173: 1 more file to edit
Entering Ex mode.  Type "visual" to go to Normal mode.
:args
[checklanguage] ru.po
:
```
As you can see, the "checklanguage" rule is passed as a dependency file.
Two variants have been tested so far:
First variant, remove "checklanguage" from dependencies and write the environment variable check in the command block.
```
cleanup-po: $(LANGUAGE).po
	if NOT DEFINED LANGUAGE (echo Set the environment variable ^%LANGUAGE^%. See README_mvc.txt. && exit 1)
	$(VIM) -u NONE -e -X -S cleanup.vim -c wq $**
```
The second variant is to remove the "$**" macro from the Vim run argument, and directly specify the dependency as a run argument.
```
cleanup-po: checklanguage $(LANGUAGE).po
	$(VIM) -u NONE -e -X -S cleanup.vim -c wq $(LANGUAGE).po
```